tests/pass0/t05_export_global.test

```diff
@@ -0,0 +1,14 @@
+discard """
+  output: "(Done 100)"
+"""
+(TypeDefs
+  (ProcTy (Int 8)))
+(GlobalDefs
+  (GlobalDef (Int 8) (IntVal 100)))
```

Comment:
```suggestion
  (GlobalDef (Int 8) (IntVal 200)))
```

I know we're not outputting this because it's the proc's return that's being checked, but I'd recommend using different numbers, so it's clearer at first glance what's happening